frontend/public/locales/en/renew-ita.json

```diff
@@ -1,4 +1,29 @@
 {
+  "contact-information": {
+    "page-title": "Contact information",
+    "phone-header": "Phone number",
+    "add-phone": "Adding a phone number helps Government of Canada contact you if there is an issue with your application. Otherwise, we'll contact you by mail.",
```

Comment:
```suggestion
    "add-phone": "Adding a phone number helps Government of Canada contact you if there is an issue with your renewal. Otherwise, we'll contact you by mail.",
```